packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts

```diff
@@ -436,7 +436,8 @@ export async function getQuestionsForAddResource(
   const addQuestion = createAddAzureResourceQuestion(
     alreadyHaveFunction,
     alreadyHaveSQL,
-    alreadyHaveAPIM
+    alreadyHaveAPIM,
+    false // TODO: to support Key Vault plugin in fx-core v2
```

Comment:
@jayzhang @1yefuwang1  
KeyVault plugin is implemented on v1 api. And I have added a param `alreadyHavekeyVault` for `createAddAzureResourceQuestion`. So just pass `false` for quick workaround.